pkg/crcontrollers/reservation/reservationcontroller.go

```diff
@@ -43,6 +45,17 @@ func NewController(client *k8s.KubeClient, log *logrus.Logger, sequentialLVGRese
 func (c *Controller) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&acrcrd.AvailableCapacityReservation{}).
+		WithOptions(controller.Options{
+			// Rater controls timeout between reconcile attempts (if result.Requeue is true)
+			// Default RateLimiter is exponential, which leads to increasing timeout after 100 retries to 5+ mins
+			// Many attempts are expected during processing ACRs with LVG Volumes, so we need to use linear RateLimiter
+			// Here we make first 30 retries with 1.5 sec timeout and then it will be 12 sec forever
+			RateLimiter: workqueue.NewItemFastSlowRateLimiter(
+				1500*time.Millisecond, // fastTimeout
```

Comment:
I think these parameters must be configurable - they depend on number of volumes and replicas

Reply:
Fixed - https://github.com/dell/csi-baremetal-operator/pull/111